js-functions/js-functions_pizza-analyzer/js/index.js

```diff
@@ -8,19 +9,58 @@ const outputSection = document.querySelector('[data-js="output-section"]');
 
 const output = document.querySelector('[data-js="output"]');
 
+
+
+
 pizzaInput1.addEventListener("input", () => {
   // write your code here
+// console.log('hello')
+const pizzaSize1 = pizzaInput1.value;
+const pizzaSize2 = pizzaInput2.value;
+
+console.log(pizzaSize1)
+calculatePizzaGain(pizzaSize1, pizzaSize2)
+updatePizzaDisplay(pizza1, pizzaSize1)
+updateOutputColor(pizzaSize1, pizzaSize2)
+
 });
 
 pizzaInput2.addEventListener("input", () => {
   // write your code here
+  const pizzaSize1 = pizzaInput1.value;
+  const pizzaSize2 = pizzaInput2.value;
+console.log(pizzaSize2)
+calculatePizzaGain(pizzaSize1, pizzaSize2)
+updatePizzaDisplay(pizza2, pizzaSize2)
+updateOutputColor(pizzaSize1, pizzaSize2)
 });
 
 // Task 1
 // define the function calculatePizzaGain here
+function calculatePizzaGain(diameter1,diameter2){
+  const area1 = Math.PI * (diameter1/2) ** 2; 
+  const area2 = Math.PI * (diameter2/2) ** 2;
+  const pizzaGain = (area2 - area1) / area1 * 100;
+  const pizzaGainFullNumber = Math.round(pizzaGain);
+  output.textContent = pizzaGainFullNumber;
+ // or -> output.textContent = Math.round(pizzaGain);
+}
+calculatePizzaGain(pizzaSize1, pizzaSize2)
 
 // Task 2
 // define the function updatePizzaDisplay here
+function updatePizzaDisplay(pizzaElement, newSize){
+  const newDisplaySize = newSize / 24 * 100;
```

Comment:
its so hard to read code without seeing it executed but your code is so well written that i can do it! Bravo